src/main/java/techreborn/items/ManualItem.java

```diff
@@ -45,9 +47,15 @@ public ManualItem() {
 	@Override
 	public TypedActionResult<ItemStack> use(final World world, final PlayerEntity player, final Hand hand) {
 		if (player instanceof ServerPlayerEntity serverPlayerEntity) {
-			NetworkManager.sendToPlayer(ClientboundPackets.createPacketOpenManual(), serverPlayerEntity);
+			PatchouliAPI.get().openBookGUI(serverPlayerEntity, new Identifier("techreborn:techreborn"));
```

Comment:
This will crash when patchouli is not present.

This should send a packet as I expect `openBookGUI` is client only, this breaks the refactors I made to help work towards: https://github.com/TechReborn/TechReborn/pull/2846

Reply:
I'll look into fixing this if this concept is something we want